CatAnalyzer/plugins/dileptonCommon.cc

```diff
@@ -198,16 +211,16 @@ void dileptonCommon::setBranchCommon(TTree* tr, int sys) {
   tr->Branch("pseudottbar", "TLorentzVector", &b_pseudottbar);
   tr->Branch("pseudottbar_dphi", &b_pseudottbar_dphi, "pseudottbar_dphi/F");
 
-  tr->Branch("partonjet1", "TLorentzVector", &b_pseudojet1);
-  tr->Branch("partonjet1_CSVInclV2", &b_pseudojet1_CSVInclV2, "partonjet1_CSVInclV2/F");
-  tr->Branch("partonjet2", "TLorentzVector", &b_pseudojet2);
-  tr->Branch("partonjet2_CSVInclV2", &b_pseudojet2_CSVInclV2, "partonjet2_CSVInclV2/F");
-  tr->Branch("partontop1", "TLorentzVector", &b_pseudotop1);
-  tr->Branch("partontop2", "TLorentzVector", &b_pseudotop2);
-  tr->Branch("partonnu1", "TLorentzVector", &b_pseudonu1);
-  tr->Branch("partonnu2", "TLorentzVector", &b_pseudonu2);
-  tr->Branch("partonttbar", "TLorentzVector", &b_pseudottbar);
-  tr->Branch("partonttbar_dphi", &b_pseudottbar_dphi, "partonttbar_dphi/F");
```

Comment:
Here "parton*" were pointing to "b_pseudo*". So the partonjet* and pseudojet* are pointing to the same object, results from the pseudotop-tuned kinematic reconstruction, up to event selection and random number effect.